doc/source/pbt.rst

```diff
@@ -1,33 +0,0 @@
-Population Based Training
```

Comment:
I would reconsider changing these pages; they're ranked quite highly on google probably because of their title.

Reply:
I'll add redirects for these pages in readthedocs.io to point to the corresponding parts in the documentation